app/javascript/components/admin/site_settings/appearance/BrandingImage.jsx

```diff
@@ -1,18 +1,51 @@
 import React from 'react';
-import { Card } from 'react-bootstrap';
-import { CloudUploadIcon } from '@heroicons/react/outline';
+import { Card, Stack } from 'react-bootstrap';
+import { CloudUploadIcon, TrashIcon } from '@heroicons/react/outline';
 import useUpdateSiteSetting from '../../../../hooks/mutations/admin/site_settings/useUpdateSiteSetting';
+import useSiteSettings from '../../../../hooks/queries/admin/site_settings/useSiteSettings';
 import FilesDragAndDrop from '../../../shared_components/utilities/FilesDragAndDrop';
+import Logo from '../../../shared_components/Logo';
+import Modal from '../../../shared_components/modals/Modal';
+import DeleteBrandingImageModal from './DeleteBrandingImageModal';
 
 export default function BrandingImage() {
   const updateSiteSetting = useUpdateSiteSetting('BrandingImage');
+  const { data: siteSettings } = useSiteSettings();
```

Comment:
With this setup, the `SiteSettings` fetches the settings, then `BrandingImage` re-fetches those settings ending with the `Logo` component fetching the `BrandingImage` setting,
We should simplify this in the future,

Reply:
Yeah, it's weird that siteSetting.BrandingImage and siteSettings.BrandingImage doesn't return the same thing.
I will remove the filename as it's not needed.

Reply:
Yup, that is why we need to come back to this so that a service will handle that properly and be the source of truth ensuring consistency,

Reply:
After some talks with @farhatahmad , we will be dropping the destroy BrandingImage completely
That solves all our problems for now.